recipes/libsimpleitk/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "libsimpleitk" %}
+{% set version = "1.2.2" %}
+{% set sha256 = "0af7387bd42bb535572b2c6c3b4ba0f9211e82a045d40ec2df507b73d5781231" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimpleITK/SimpleITK/releases/download/v{{version}}/SimpleITK-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [ win and py27 ]
+
+requirements:
+  build:
+    - cmake   >=3.11
+    - ninja
+    - lua     5.1.*          # [ not win ]
+    - swig                   # [ win ]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - libitk  <5
+
+  run:
+    - vs2015_runtime  # [win]
+    - {{ pin_compatible('libitk', max_pin='x.x.x') }}
+
+test:
+  requires:
+    - cmake
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  source_files:
+    - Examples
+  commands:
+    - test -d $PREFIX/include/SimpleITK*                                 # [not win]
+    - if not exist %LIBRARY_INC%\\SimpleITK* exit 1                      # [win]
+    - cmake -D "CMAKE_SYSTEM_PREFIX_PATH:FILEPATH=${PREFIX}" ./Examples  # [not win]
+    - cmake --build . --config Release                                   # [not win]
+
+about:
+  home: http::/www.simpleitk.org
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: Development headers and libraries for SimpleITK a simplified interface to the Insight Toolkit.
+
+extra:
+    recipe-maintainers:
+      - blowekamp
+      - zivy
```

Comment:
@zivy Are you OK with being a maintainer?

Reply:
I'm ok with being a maintainer.